src/query/ast/src/ast/statements/connection.rs

```diff
@@ -48,8 +48,8 @@ impl Display for CreateConnectionStmt {
         }
         write!(f, "{} ", self.name)?;
         write!(f, "STORAGE_TYPE = {} ", self.storage_type)?;
-        for (k, v) in &self.storage_params {
-            write!(f, "{} = {} ", k, v)?;
+        for k in &self.storage_params.keys() {
+            write!(f, "{} = ******** ", k)?;
```

Comment:
Prefre to use https://github.com/datafuselabs/databend/blob/main/src/common/base/src/base/string.rs#L101
`mask_string(v, 3)`

Reply:
done